packages/stream_chat_flutter/lib/src/message_input/stream_message_send_button.dart

```diff
@@ -10,7 +11,9 @@ class StreamMessageSendButton extends StatelessWidget {
     super.key,
     this.timeOut = 0,
     this.isIdle = true,
+    @Deprecated('Will be removed in the next major version')
     this.isCommandEnabled = false,
+    @Deprecated('Will be removed in the next major version')
     this.isEditEnabled = false,
```

Comment:
Should we document the alternative they have to use here?
You deprecated them, but they already stopped having any effect right?